_help/get-started/overview.md

```diff
@@ -24,5 +26,8 @@ redirect_from:
 - /help/creating-an-account/why-do-i-need-to-confirm-my-email-address-and-my-phone-number/
 - /help/creating-an-account/why-do-i-need-to-use-logingov-to-access-government-services-online/
 - /help/creating-an-account/why-is-my-confirmation-link-invalid/
-redirect_to: /help
+- /help/get-started/
 ---
+
+# {% t page.title %}
```

Comment:
I had been including the page.title inside the content of the text for the help pages I wrote... this is a good reminder I should go back and fix those